### PR TITLE
[SDP-1056] sdp: add SDP Admin API spec

### PR DESCRIPTION
**Pull request by CaioTeixeira95**
This PR adds the API spec for the SDP Admin APIs.

### CODE REVIEW

File: openapi/stellar-disbursement-platform/main.yml [hunk 1]
```diff
@@ -1831,6 +1831,180 @@ paths:
         - application/json
       security:
         - BearerAuth: []
+  
+  /tenants:
+    get:
+      summary: Get All Tenants
+      responses:
+        "200":
+          description: "This endpoint returns all registered tenants on the system"
```
Comment (marwen-abid) [inline openapi/stellar-disbursement-platform/main.yml:1840]:
```suggestion
          description: "A list of provisioned tenants with their configurations"
```

File: openapi/stellar-disbursement-platform/main.yml [hunk 2]
```diff
@@ -1831,6 +1831,180 @@ paths:
...
+                type: string
+                enum: [ DRY_RUN, AWS_SMS, TWILIO_SMS ]
+                description: The SMS sender provider.
+              base_url:
+                type: string
+                description: The SDP backend server's base URL.
+              sdp_ui_base_url:
+                type: string
+                description: The SDP UI/dashboard Base URL.
+              distribution_account:
```
Comment (marwen-abid) [inline openapi/stellar-disbursement-platform/main.yml:1900]: I see that `TenantRequest` takes a `distrubution_account` but I don't see that value being used anywhere. Please double check that

File: openapi/stellar-disbursement-platform/main.yml [hunk 3]
```diff
@@ -2736,6 +2975,9 @@ securityDefinitions:
     name: "Authorization"
     in: "header"
     description: "JWT used for authentication, signed with an ES256 private key."
+  BasicAuth:
+    type: "http"
+    scheme: "basic"
```
Comment (marwen-abid) [inline openapi/stellar-disbursement-platform/main.yml:2980]: We can add a description here. 
```
description: "Basic HTTP Authentication with a base64-encoded string of 'ADMIN_ACCOUNT:ADMIN_API_KEY'."
```

**Review by marwen-abid — APPROVED** [review]
LGTM! left a couple of comments.

File: openapi/stellar-disbursement-platform/main.yml (hunk 2 above)
Comment (CaioTeixeira95) [reply openapi/stellar-disbursement-platform/main.yml:1900]: That's true, I believe we missed adding it to the tenant after the tenant provision.

Comment (CaioTeixeira95) [reply openapi/stellar-disbursement-platform/main.yml:1900]: As discussed on the stand-up, the field was removed.

**Pull request merged**